ttnn/cpp/ttnn/operations/moreh/moreh_getitem/moreh_getitem_pybind.cpp

```diff
@@ -0,0 +1,23 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "moreh_getitem_pybind.hpp"
+
+#include "pybind11/decorators.hpp"
+#include "ttnn/operations/moreh/moreh_getitem/moreh_getitem.hpp"
+
+namespace ttnn::operations::moreh::moreh_getitem {
+void bind_moreh_getitem_operation(py::module& module) {
+    bind_registered_operation(
+        module,
+        ttnn::moreh_getitem,
+        "Moreh moreh_getitem operation",
+        ttnn::pybind_arguments_t{
+            py::arg("input"),
+            py::arg("index_tensors"),
+            py::arg("index_dims"),
+            py::arg("output") = std::nullopt,
+            py::arg("output_mem_config") = std::nullopt});
```

Comment:
>In ttnn we use kw_only arguments dtype and memory_config.
please adjust for consistency.
Also make sure that arguments which are kw args in torch are also kw args here.
Thank you!


https://github.com/tenstorrent/tt-metal/pull/12294#discussion_r1750369085